ground/src/main/java/com/google/android/ground/model/mutation/LocationOfInterestMutation.kt

```diff
@@ -29,9 +29,10 @@ data class LocationOfInterestMutation(
   override val retryCount: Long = 0,
   override val lastError: String = "",
   val jobId: String = "",
+  val customId: String? = null,
```

Comment:
Can we default to empty string and make non-null?

Reply:
Done